wagtail/admin/views/home.py

```diff
@@ -144,3 +146,10 @@ def default(request):
     redirected to the login page.
     """
     raise Http404
+
+
+@cache_page(31536000)  # 1 year
+def sprite(request, **kwargs):
+    response = render(request, "wagtailadmin/icons/sprite.svg")
+    response["Content-Type"] = "image/svg+xml"
```

Comment:
Since we're embedding this onto the page with .innerHTML, I think we should change this MIME type to text/html, or change that embed code to use an <img> tag instead. This makes it clear what the format is since HTML has some extra security concerns that we don't usually need to consider for SVG.

Reply:
Good catch! I believe this is an artefact of earlier attempts. I tried to implement SVG as an external file.

I will drop this line. A Django view default type is `text/html`.

Reply:
Thanks! It might also be worth changing the template name from `.svg` to `.html` to make it clear there too